src/redismodule.h

```diff
@@ -1671,6 +1682,7 @@ static int RedisModule_Init(RedisModuleCtx *ctx, const char *name, int ver, int
     REDISMODULE_GET_API(RdbStreamFree);
     REDISMODULE_GET_API(RdbLoad);
     REDISMODULE_GET_API(RdbSave);
+    REDISMODULE_GET_API(SetOperate);
```

Comment:
Is it better to move it up after the datatype block? .e.g `REDISMODULE_GET_API(StreamTrimByID);`